cms/schemas/documents/song.js

```diff
@@ -66,7 +66,13 @@ const song = {
       options: {
         source: "title",
         slugify: (input) =>
-          input.toLowerCase().replace(/\s+/g, "-").slice(0, 200),
+          input
+            .toLowerCase()
+            .replace(/\s+/g, "-")
+            .replace("å", "aa")
+            .replace("æ", "ae")
+            .replace("ø", "oe")
```

Comment:
```suggestion
            .replace("å", "aa")
            .replace("æ", "ae")
            .replace("ø", "oe")
            .replace("[^a-z]", "")
```

Legge til en liten ekstra regex kanskje?